docs/CHANGELOG.asciidoc

```diff
@@ -68,6 +68,440 @@ Review important information about the {kib} 7.x releases.
+
+* The following functions are deprecated and won't be part of the 8.0.0 public API:
+** `convertRangeFilterToTimeRangeString`
+** `extractTimeFilter`
+** `changeTimeFilter`
+** `mapAndFlattenFilters`
+
+On the server side, `esFilters` is now deprecated. All utility functions are available for import from `@kbn/es-query`.
+
+For more information, refer to {kibana-pull}106566[#106566].
+
+*Impact* +
+Import all utility functions from `@kbn/es-query`.
+====
+      
+[discrete]
+[[deprecation-106232]]
+.Refactors `textarea` UI argument
+[%collapsible]
+====
+*Details* +
+Based on the `x-pack/plugins/canvas/CONTRIBUTING.md`, `recompose` has been removed in favor of React hooks at `textarea`. For more information, refer to {kibana-pull}106232[#106232].
+
+*Impact* +
+There is no user-facing impact.
+====
+      
+[discrete]
+[[deprecation-105981]]
+.Deprecates `xpack.maps.showMapVisualizationTypes`
+[%collapsible]
+====
+*Details* +
+Deprecates the `xpack.maps.showMapVisualizationTypes` *Maps* setting. For more information, refer to {kibana-pull}105981[#105981].
+
+*Impact* +
+When you upgrade, remove `xpack.maps.showMapVisualizationTypes` from your kibana.yml file.
+====
+      
+[discrete]
+[[deprecation-105742]]
+.Refactors `string` UI argument
+[%collapsible]
+====
+*Details* +
+Based on the `x-pack/plugins/canvas/CONTRIBUTING.md`, `recompose` has been removed in favor of React hooks at `string`. For more information, refer to {kibana-pull}105742[#105742].
+
+*Impact* +
+There is no user-facing impact.
+====
+      
+[discrete]
+[[deprecation-104685]]
+.Deprecates legacy audit logger
+[%collapsible]
+====
+*Details* +
+The legacy audit logger is deprecated. For more information, refer to {kibana-pull}104685[#104685].
+
+*Impact* +
+Use the ECS-comliant audit logger. For more information, refer to <<audit-logging-settings,Audit logging settings>>.
+====
+      
+[discrete]
+[[deprecation-100781]]
+.Deprecates scripted fields
+[%collapsible]
+====
+*Details* +
+Adding scripted fields to index patterns is deprecated. For more information, refer to {kibana-pull}100781[#100781].
+
+*Impact* +
+Use runtime fields. For more information, refer to <<runtime-fields,Explore your data with runtime fields>>.
+====
+      
+[float]
+[[features-7.15.0]]
+=== Features
+{kib} 7.15.0 adds the following new and notable features.
+
+APM::
+* Enables metric-powered UI {kibana-pull}104929[#104929]
+Data ingest::
+* Adds copy_from to set processor {kibana-pull}104070[#104070]
+* Adds community id processor {kibana-pull}103863[#103863]
+* Adds network direction processor {kibana-pull}103436[#103436]
+Elastic Security::
+For the Elastic Security 7.15.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Lens & Visualizations::
+* Synchronizes cursor position for x-axis across all *Lens* visualizations on a dashboard {kibana-pull}106845[#106845]
+* Client Side caching in *Visualize* {kibana-pull}105589[#105589]
```

Comment:
We should probably rewrite this as "Adds client-side caching in Visualize to eliminate unnecessary data fetching when editing visualizations."